site/source/tutorials/008_animations/index.md

```diff
@@ -0,0 +1,370 @@
+					delay,
+				},
+				controls: {
+					play: this._playHearts,
+					onFinish: sequence === this._numHearts -1 ? this._onHeartsFinish : undefined
+				}
+		},
+		{
+			id: `${id}Scale`,
+			effects: [
+				{ transform: 'scale(1)' },
+				{ transform: 'scale(0.8)' },
+				{ transform: 'scale(1)' },
+				{ transform: 'scale(1.2)' },
+				{ transform: 'scale(1)' }
+			],
+			timing: {
+				duration: 500,
+				iterations: Infinity,
+				delay,
+				easing: 'ease-in-out'
+			},
+			controls: {
+				play: this._playHearts
+			}
+		}
+	];
+}
+
+private _getHearts(): HNode[] {
+	const hearts = [];
+	let play = false;
+	let i;
+	for (i = 0; i < this._numHearts; i++) {
+		const key = `heart${i}`;
+		hearts.push(v('div', { classes: css.heart, key }));
+		this.meta(WebAnimation).animate(key, this._getHeartAnimation(key, i, play));
+	}
+	return hearts;
+}
+
+// add this as the last child of the `root` HNode in render
+v('div', { classes: css.heartsHolder }, this._getHearts())
+
+```
+
+Now you should see the hearts appearing and floating away when the zombies get to the middle of the screen.
+
+{% section %}
+
+## Controlling our animations
+
+Due to the reactive nature of the Dojo 2 `WebAnimation` meta, we can control our animation by changing the properties passed. For example we can change the `playbackRate` programatically.
+Lets go ahead and add slider widgets to our page to change the speed of the zombie shuffle and legs.
+
+``` typescript
+// add the import, you may need to npm install `@dojo/widgets`
+import Slider from '@dojo/widgets/slider/Slider';
+
+// add the variable and then change event
+private _zombieLegsPlaybackRate = 1;
+
+private _onZombieLegsPlaybackRateChange(event: Event) {
+	const value = (event.target as HTMLInputElement).value;
+
+	this._zombieLegsPlaybackRate = parseFloat(value);
+	this.invalidate();
+}
+
+// use the new this._zombieLegsPlaybackRate property in the animation
+private _getZombieLegAnimation(id: string, front?: boolean): AnimationProperties {
+	const effects = [
+		{ transform: 'rotate(0deg)' },
+		{ transform: 'rotate(-5deg)' },
+		{ transform: 'rotate(0deg)' },
+		{ transform: 'rotate(5deg)' },
+		{ transform: 'rotate(0deg)' }
+	];
+
+	if (front) {
+		effects.reverse();
+	}
+
+	return {
+		id,
+		effects,
+		timing: {
+			duration: 1000,
+			iterations: Infinity
+		},
+		controls: {
+			play: this._play,
+			playbackRate: this._zombieLegsPlaybackRate // add it here
+		}
+	};
+}
+
+// finally, add the control to the top of the render function
+v('div', { classes: css.controls }, [
+	w(Slider, { min: 0.1, max: 10, step: 0.1, value: this._zombieLegsPlaybackRate, onChange: this._onZombieLegsPlaybackRateChange })
+])
+```
+
+Open your browser and you should now be able to speed up and slow down the zombie leg animation as they move across the screen.
+
+{% section %}
+
+## Summary
+
+In this tutorial, we learned:
+
+* How to use a Dojo 2 meta
+* How to pass AnimationProperties via the WebAnimation meta
```

Comment:
How to pass AnimationProperties via the WebAnimation meta -> How to pass `AnimationProperties` via the `WebAnimation` meta